vending-app/src/main/java/org/microg/vending/billing/core/HttpClient.kt

```diff
@@ -79,7 +79,7 @@ class HttpClient(context: Context) {
             override fun getHeaders(): Map<String, String> = headers
             override fun getBody(): ByteArray = payload.encode()
             override fun getBodyContentType(): String = "application/x-protobuf"
-        }.setShouldCache(cache).setRetryPolicy(DefaultRetryPolicy(POST_TIMEOUT, 0, 0.0F)))
+        }.setShouldCache(cache).setRetryPolicy(DefaultRetryPolicy(POST_TIMEOUT, 3, 0.0F)))
```

Comment:
This is likely to break things. If the first request goes through but timeouts at the reply, a retry is going to fail. Retries can only succeed if they happen because the timeout happens before sending the request (e.g. at DNS or before TLS termination).